### PR TITLE
Basic direct Model usage documentation

### PR DESCRIPTION
**Pull request by Gnurro**
New .MD covering directly using backends.Model for generation.
Also comes with a small change to backends.__init__.py to force loading the model registry JSON as UTF-8 - to allow simple one-line loading of the model registry in howto_basic_backend_usage.md.

### CODE REVIEW

File: docs/howto_basic_backend_usage.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+content. The method returns the full prompt, full output and generated text as a tuple of strings. See 
+[the model addition howto](howto_add_models.md) for more information.
+## Basic example
+Load a supported model and generate a reply:
+```python
+import backends
+import os
+
+# path to the model registry file, this assumes this script is in the clembench root directory:
+_model_registry_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "backends", "model_registry.json")
```
Comment (phisad) [inline docs/howto_basic_backend_usage.md]: Just one little thing: This line is actually done internally when the path is not given. So do we want to be that verbose here? If not, then we should also briefly mention  `load_custom_registry`

Comment (davidschlangen) [reply docs/howto_basic_backend_usage.md]: I agree, this is probably too much information, if the code can run without it, just by assuming that everything has been set up as per the default.

Comment (Gnurro) [reply docs/howto_basic_backend_usage.md]: I tried it without specifying the path like that and it did not work on my local machine. This also shows how to set a custom path for cases where someone might use this clembench functionality from any directory.

Using `load_custom_registry` goes beyond the very basic usage this documentation is supposed to cover.

Comment (Gnurro) [reply docs/howto_basic_backend_usage.md]: https://github.com/clp-research/clembench/blob/cf63294e27d578aba75ac1807b8de7dffe12de33/backends/__init__.py#L247
Relying on this did not work in my testing, as `project_root` is a) the wrong path on my system after a basic import of `backends`, and b) if set as it is in `backends.__init__.py` (also an dditional line that needs to be in the example code), it is the drive root on my system, not the clembench directory root (as python paths are always relative to the CWD), so it wouldn't work either.

Comment (Gnurro) [reply docs/howto_basic_backend_usage.md]: I checked again and it works now as intended, without the extra path. Something seems to have been off with my IDE, which apparently got fixed by restarting it.
I'd appreciate if someone could test this in a different environment to make sure. Other than that, this should resolve this.

**Review by sherzod-hakimov — APPROVED** [review]

**Pull request merged**